TASK: update for neos 5 compatibility

andrehoffmann30 opened the pull request.

andrehoffmann30 committed:
TASK: replace SystemLoggerInterface with LoggerInterface

andrehoffmann30 committed:
TASK: add neos 5.0 restriction to composer.json

daniellienert committed:
TASK: Replace ObjectManager with EntityManagerInterface

dfeyer commented:
Can you explain why this is WIP ? Thanks

andrehoffmann30 commented:
We started to update, but we still need to test the changes.

dlubitz commented on Classes/Command/ImportCommandController.php:
```diff
@@ -298,7 +304,7 @@ protected function executeCommand(PresetPartDefinition $partSetting)
             return $count;
         } catch (\Exception $exception) {
-            $this->logger->logException($exception);
+            $this->throwableStorage->logThrowable($exception);
```
Shouldn't it be:
```
$logMessage = $this->throwableStorage->logThrowable($exception);
$this->logger->error($logMessage, LogEnvironment::fromMethodName(__METHOD__));
```

andrehoffmann30 replied:
I replaced all logThrowable calls with your suggestion @dlubitz  Thank you.

andrehoffmann30 commented on Classes/Aspect/EventLogAspect.php:
```diff
@@ -74,8 +76,8 @@ public function flushEvents(JoinPointInterface $joinPoint)
-        } catch (\Exception $exception) {
-            $this->logger->logException($exception);
+        } catch (Exception $exception) {
+            $this->throwableStorage->logThrowable($exception);
```
Must be included into a logger call

fnkr committed:
FIX: Use PSR LogLevel classes

andrehoffmann30 committed:
TASK: adjust use of logThrowable function to log exceptions

andrehoffmann30 commented:
Hi @dfeyer, @dlubitz I updated this branch and fixed the last issues. In my Opinion, it is read to be merged. We are using this branch in two neos 5 projects and don´t have any problems so far.

daniellienert committed:
BUGFIX: Inject missing logger
